.github/workflows/setup-action-host.yml

```diff
@@ -0,0 +1,76 @@
+  name: Setup GitHub Action Host Workflow
+
+  on:
+    pull_request:
+      branches:
+        - main
+        - release-1.x
```

Comment:
This Action seems to be called within `linux-browser-compatibility-test.yml`, do we still need this trigger?

Reply:
This file has been changed.